README.md

```diff
@@ -41,8 +41,8 @@ vars:
     apple_search_ads_database: your_database_name
     apple_search_ads_schema: your_schema_name 
 ```
-### Disabling models
-It's possible that your Apple Search Ads connector does not sync every table that this package expects. If your syncs exclude certain tables, it is because you either don't use that functionality in Apple Search Ads or actively excluded some tables from your syncs. To disable the corresponding functionality in the package, you must add the relevant variables. By default, all variables are assumed to be true. Add variables for only the tables you would like to disable. 
+### Disable models
+It's possible that your Apple Search Ads connector does not sync every table that this package expects. If your syncs exclude certain tables, it is because you either don't use that functionality in Apple Search Ads or actively excluded some tables from your syncs. To disable the corresponding functionality in the package, you must add the relevant variables. By default, the package assume that all variables are true. Add variables for only the tables you want to disable. 
 
 The `apple_search_ads_using_search_terms` variable below refers to the `search_terms_report` table. The [search match](https://searchads.apple.com/help/campaigns/0006-understand-search-match) functionality must be turned on within specified ad groups in order for there to be data populated in this table. 
```

Comment:
@fivetran-joemarkiewicz I dont understand what we mean by "specified ad groups" here. Which specified ad groups?

Reply:
Hey @fivetran-chloe the Apple Search Ads UI allows for you to create multiple ad groups and you must enable the `search_terms` functionality within each distinct ad group.

We are trying to highlight the fact that you must enable `search_terms` within each ad group if you want the data to be populated.

Reply:
Got it - thanks!

Reply:
Looks great, thanks!